server/routerlicious/packages/services-core/package.json

```diff
@@ -32,11 +32,11 @@
 		"typetests:prepare": "flub typetests --dir . --reset --previous --normalize"
 	},
 	"dependencies": {
-		"@fluidframework/common-utils": "^3.1.0",
 		"@fluidframework/gitresources": "workspace:~",
 		"@fluidframework/protocol-definitions": "^3.2.0",
 		"@fluidframework/server-services-client": "workspace:~",
 		"@fluidframework/server-services-telemetry": "workspace:~",
+		"@types/events": "^3.0.0",
```

Comment:
Same as in protocol-base; not sure it's necessary. Brought it because it's in common-utils.